pkg/gofr/container/health_test.go

```diff
@@ -9,7 +9,6 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-
```

Comment:
you should not remove this line

I'm surprised gci linter didn't report an issue

https://github.com/gofr-dev/gofr/blob/eedaec521dac7b4d624b7f4cab510e8eab9fde11/.golangci.yml#L10-L14